tests/modules/assets/action-get-measures.test.ts

```diff
@@ -0,0 +1,99 @@
+import {
+  ApiAssetGetMeasuresRequest,
+  ApiAssetGetMeasuresResult,
+} from "../../../index";
+
+import { beforeEachTruncateCollections } from "../../hooks/collections";
+import { beforeAllCreateEngines } from "../../hooks/engines";
+import { beforeEachLoadFixtures } from "../../hooks/fixtures";
+
+import { sendDummyTempPayloads, useSdk } from "../../helpers";
+
+jest.setTimeout(10000);
+
+describe("AssetsController:getMeasures", () => {
+  const sdk = useSdk();
+
+  beforeAll(async () => {
+    await sdk.connect();
+    await beforeAllCreateEngines(sdk);
+  });
+
+  beforeEach(async () => {
+    await beforeEachTruncateCollections(sdk);
+    await beforeEachLoadFixtures(sdk);
+  });
+
+  afterAll(async () => {
+    sdk.disconnect();
+  });
+
+  it("should returns measures linked to the asset", async () => {
```

Comment:
nit: split this test in three independant tests. You include all three in another describe function, with a beforeEach:

  ```
  await sendDummyTempPayloads(sdk, [
      { deviceEUI: "unlinked1", temperature: 12 },
      { deviceEUI: "unlinked1", temperature: 14 },
      { deviceEUI: "linked1", temperature: 20 },
      { deviceEUI: "linked1", temperature: 22 },
      { deviceEUI: "linked1", temperature: 24 },
      { deviceEUI: "linked2", temperature: 42 },
      { deviceEUI: "linked2", temperature: 44 },
    ]);
    await sdk.collection.refresh("engine-ayse", "measures");
```

Reply:
The 3 things I'm testing are actually related to the same context, I just need to check 3 documents. I used the extra blocks because otherwise I cannot redeclare the `result` variable